src/utils/location.js

```diff
@@ -0,0 +1,30 @@
+const decode = decodeURIComponent;
+
+// Copied from vue-router utils
```

Comment:
Zastanawiam się czy to jest konieczne, jeśli to jest już w tym routerze :p

Reply:
Jest konieczne, ponieważ ta funkcja nie jest eksportowana na zewnatrz, patrz źródło `vue-router`a:

https://github.com/vuejs/vue-router/blob/ea8cb474f869a5a12a095fcb5989c45c68971d14/src/util/query.js#L37-L63